tasks/sync-granule/index.js

```diff
@@ -89,7 +92,6 @@ exports.syncGranule = function syncGranule(event) {
   return download(ingest, downloadBucket, provider, input.granules)
     .then((granules) => {
       if (ingest.end) ingest.end();
```

Comment:
I did not add anything here.  This seems straighforward.  Should I have?

Reply:
The thing that would be useful is to know that we got to the end of ALL the task code, so we can narrow down if it is the message adapter or not.

Reply:
Another thing that would help is in the tests where we're waiting on a second workflow that was kicked off, the arn of the workflow we're waiting for. Will save some debug time.

Reply:
I'm not sure which other workflows you're thinking of.  We wait for one of the ingestGranules and I log that here https://github.com/nasa/cumulus/pull/525/files#diff-9536f9b4e584de6b5cab1794daa824b9R81 .   Were you thinking of getting all of the kicked-off arns?

Reply:
Sorry, my bad. I missed that. You're right, that's good.